update-completed-sprint-on-issue-closed/workflow.yml

```diff
@@ -0,0 +1,90 @@
+name: Update CompletedSprint on Issue Closed
```

Comment:
❗ Afaik callable workflows have to live in `.github/workflows` unfortunately. Has that changed? What I did in each of the other workflows in here is put a symlink to the folder so that work developing it's still located in a folder with its own readme.

Reply:
Ah yeah, i just forgot to link it. I am debating putting it into a single (empty) repo to test it manually, and moving issues around there.